src/main.cpp

```diff
@@ -30,6 +30,7 @@
 #include "glacierwindowmodel.h"
 #include <QScreen>
 
+
```

Comment:
should delete..

Reply:
it need for line 65

Reply:
I think eekkelund means the blank line here